Cliqz/Search/CliqzSearchViewController.swift

```diff
@@ -192,10 +192,12 @@ class CliqzSearchViewController : UIViewController, KeyboardHelperDelegate, UIAl
             self.privateMode = privateMode
             Engine.updateExtensionPreferences(privateMode: privateMode)
             if privateMode && privateModeOverlay.superview == nil {
+                #if !PAID
```

Comment:
why it's removed for Lumen? then the whole if doesn't make sense. You should wrap the whole if block under the macro. do we need this overlay on other products?